.eslintrc.json

```diff
@@ -59,6 +59,11 @@
             "error",
             "always"
         ],
+        "eol-last": [
+            "error",
+            "always"
+        ],
+        "no-multiple-empty-lines": "error",
```

Comment:
I would suggest adding the `max: 1` setting here, otherwise you could also add multiple empty lines... Or do you think that's a good idea at anytime?